### PR TITLE
feature/coupled-crow: Updates for consistency 

### PR DESCRIPTION
**Pull request by JessicaMeixner-NOAA**
This partially addresses issue #172.  This updates to use the ufs-weather-model instead of ufs-s2s-model and updates the hash the model is pointing to. 

This consolidates the MOM_inputs so there is only one per resolution and the configurable variables are consistent with the RT tests for the ufs-weather-model

### CODE REVIEW

**Comment by JessicaMeixner-NOAA** [comment]
This also points to a new fix directory which added fractional grid inputs (not using them yet), sync'd with the latest develop branch global-workflow fix directory, including making a new directory for the wave fix folder.  This also uses resolution dependent fix directories for CICE and MOM6.

**Review by jiandewang — APPROVED** [review]
made test runs on orion and hera, both succeed

**Comment by JessicaMeixner-NOAA** [comment]
Thanks @jiandewang  for running a test & reviewing.  @JianKuang-NOAA or @WalterKolczynski-NOAA this needs a review from someone who has write permission before it can be merged.

**Review by jkuang-exploro — APPROVED** [review]

**Pull request merged**
